### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: app/players/dwarves/dwarf.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
```
Comment (Yaroslav-Stetsko) [inline app/players/dwarves/dwarf.py:5]: No need to import ABC
Elf and Dwarf should be inherited from the Player only

**Review by Yaroslav-Stetsko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
```
[the remark this answers is not in the document]
Comment (HStakhiv) [reply app/main.py:3]: Yes, but if I want to indicate that this is a list of `Player`s then it is necessary to import them

File: app/players/dwarves/dwarf.py (hunk 1 above)
Comment (HStakhiv) [reply app/players/dwarves/dwarf.py:5]: These are PyCharm's recommendations

File: app/players/dwarves/dwarf_blacksmith.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+from abc import ABC
+
+
+class DwarfBlacksmith(Dwarf, ABC):
```
Comment (Y-Havryliv) [inline app/players/dwarves/dwarf_blacksmith.py:5]: you don't need `ABC` here

File: app/players/dwarves/dwarf_warrior.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+from app.players.dwarves.dwarf import Dwarf
+from abc import ABC
+
+
+class DwarfWarrior(Dwarf, ABC):
```
Comment (Y-Havryliv) [inline app/players/dwarves/dwarf_warrior.py:5]: you don't need `ABC` here

File: app/players/elves/druid.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+from app.players.elves.elf import Elf
+from abc import ABC
+
+
+class Druid(Elf, ABC):
```
Comment (Y-Havryliv) [inline app/players/elves/druid.py:5]: you don't need `ABC` here

File: app/players/elves/elf_ranger.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+from app.players.elves.elf import Elf
+from abc import ABC
+
+
+class ElfRanger(Elf, ABC):
```
Comment (Y-Havryliv) [inline app/players/elves/elf_ranger.py:5]: you don't need `ABC` here

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**